dist/components/product/_product.dust

```diff
@@ -0,0 +1,38 @@
+{#product}
+<div class="c-product {?isHighlight}c--highlight{/isHighlight}" role="group">
+    <a class="c-product__title" href="{url}" title="{title}" rel="external">
+        <h2>{title}</h2>
+    </a>
+
+    {#image}
+        <div class="c-product__image">
+            <a href="{url}" title="{title}" rel="external">
+                <img src="{path}" alt="{title}" />
+            </a>
+        </div>
+    {/image}
+
+    <span class="c-product__price" aria-label="Price is {price}">{price}</span>
+
+    {#isDiscount}
+        <span class="c-product__price" aria-label="Old price is {price}">{oldPrice}</span>
+
+        <s class="c-product__price c--discount" aria-label="New discounted price is {price}">{discountPrice}</s>
```

Comment:
Same idea here: should the `aria-label` static text be customizable? Maybe the site language uses the term "sale" instead of "discount"